### ISSUE TITLE
Cerner Navigation option on Appointments page from Appointments Team (formerly VAOS)

### ISSUE THREAD

**Issue by ldelacosta** [department-of-veterans-affairs/va.gov-team#59293]
## About your team

- Team name: Appointments Team (formerly VAOS) 
-	OCTO-DE product owner: Katherine Lawyer 
-	Product manager: Jeff Roof and Leah De La Costa
-	Designer: Peter Russo, Melissa Lefevre and Ciera Maddox
-	FE engineer: Simi Adebowale 
- Accessibility specialist:
- Content writer: 
-	Product/team Slack channel: #appointments-team 
 
### What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [X] Making revisions to one or more existing VA.gov pages
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] Something else 
 
### Give a brief description of what your team needs support with:

My team needs support with content for when a Veteran is re-direct to Cerner portal from VA.gov 
 
 
### Will this new product be released incrementally (for instance 25% of users initially)? 
  - [X] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [ ] No
 
#### If yes, please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
### When does this work need to be completed?
- Estimated launch date: August 2023
- Estimated staging review date: July 2023
- Content, Accessibility, and IA work needed by: June 2023
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- [Product Outline](url) 
- Link to or attach user flows:
- Link to prototype or mockups:
- Link to your research plan: 
- Link to any epics/issues that may be helpful:

#### Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

 **Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.
 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by outerpress** [comment]
This is the in-app Cerner redirect similar to what other tools on MHV on VA.gov will be using. It also includes proposed changes to the same alert on the Appointments static landing page.

FYI @kristenmcconnell @mnorthuis @Cieramaddox @DanielleThierryUSDSVA @laurwill

**Comment by kristinoletmuskat** [comment]
Confirmed that we don't need to do any CAIA IA work on this ticket @RLHecht

**Comment by outerpress** [comment]
Hi @RLHecht and @coforma-terry ! Just checking in on this one, lmk if you know when the team'll be able to take a look. Thanks!

**Comment by Cieramaddox** [comment]
We are also exploring what it would look like when both the **cerner redirect alert** and the **feature to Opt Out from MHV on VA.gov** appear together [here](https://www.figma.com/file/mylP38N3rxVWIk707r0ZmN/Feature-to-opt-out-from-MHV-on-VA.gov?type=design&node-id=1277%3A30188&t=woz5Qap7TlO8NJTP-1).

**Comment by laurwill** [comment]
Hi @outerpress and @Cieramaddox , sorry for delay here — and thanks for the link to the opt out version! 

I stared leaving a few comments, and I'll have an update for you on this work early next week.

**Comment by kristenmcconnell** [comment]
I added a few comments as well.

**Comment by laurwill** [comment]
Hi @outerpress , we can talk more about this in today's session, but adding a few notes here:

**Cerner message on appointments tool landing page**
- Changes to this content need to go through public websites. And this content appears on all health tool landing pages, so changes would need coordination to align all pages.
- The "Make sure you're in the right place" wording may not work here, because they're not really inside either portal yet. We'd need to model this more on the [My VA version](https://www.figma.com/file/0wG5mpi760FGvVh89phapu/User-flow-for-MHV-Portal-to-Appointments?type=whiteboard&node-id=929-3587&t=1VkSW15VUIa8eq4m-0), that says "Choose your health management portal" and then directs people to enter one or the other. 

**Opt out message**
- Appointments is different from the other MHV on VA.gov health tools, because the key features are already on VA.gov. There isn't really a "trusted My HealtheVet appointments tool" that we're moving to a new home, in the way that we're moving secure messaging from one place to another. Let's talk more about this one!

**Comment by outerpress** [comment]
@laurwill thanks! Capturing various conversations here:

**Opt out message**
We decided at the 6/21 SME Sync not to add this to VAOS. I copied comments relevant to the Cerner redirect over to the design file

**Cerner message on static landing page**
We'll work with Public Websites to get a sense of the LOE on this work. FYI @ldelacosta 

**Cerner message on VAOS**
- @laurwill there's [one open question](https://www.figma.com/file/iqjkZx2EbrZu6leL6YfN45?node-id=1222:26182&mode=design#479392749) about the heading on the alert. 
- Once content is done, will the CAIA review on this ticket be complete, or is accessibility taking a gander as well.

**Comment by coforma-terry** [comment]
Tagging @sara-amanda  for visibility :)

**Comment by outerpress** [comment]
Thanks @coforma-terry and @sara-amanda ! We want to finalize this by Wednesday.

Our main a11y question is if it's okay to use Action links in both contexts (as shown in the design files, or if regular links would be preferable. Of course, if you see any other issues, please let us know.

**Comment by coforma-terry** [comment]
No worries @outerpress - we'll take a look at this for you and get back to you asap :)

**Comment by sara-amanda** [comment]
## CAIA Update
_Pursuant to the request for review by Wednesday, June 28, 2023, please find our notes below, relating to your revised design. Should you have any questions, please feel free to reach out, and we can assist further._ 

### Action Links
- @outerpress you should be good because both the "Go to..." are navigation, so navigation should be links and not buttons. They can be a link or action link. 
    - Your updated designs show: action links. 
    - Confirmed guidance in #55859
- [Also see Platform's guidance on **Link - Action,** here.](https://design.va.gov/components/link/action)
    - [x] Action Links replace green primary buttons that take users to another page.
    - [x] Keep Action Link content short. Start with a verb. For example: “Apply for health care benefits” or “Register for care”
    - [x] Refer to the content considerations for the link.
    - [x] Keep Action Link content short. Start with a verb. 
    - [x] Don’t use Action Links link to go to another page or site that isn’t taking users to start an action.
    - [x] Primary (green) and secondary (blue) Action Links can exist on the same page, but we don’t recommend placing them side by side. Use a primary (green) Action Link for the primary call to action on a page or the start of a digital service. Use only one primary Action link per page.
    - [x] Use a secondary (blue) Action Link when there are multiple Action Links on a page or if the actions are of equal hierarchy.

### Alerts
#### Currently Using 
- **Warning Alert:** It appears the warning alert is being used here. 
    - These are used to warn a user, such as when there are negative consequences, or when something has gone wrong. 
#### Alternative Alert Considerations
- [Also see Platform's guidance on **Alerts**, here.
](https://design.va.gov/components/alert)
- **Standard Alert:** This is used to provide helpful information or something that warrants a user’s attention. Not used for negative consequences.
- **Sign in or tool prompt**: Used to prompt a user to sign in, create an account, or launch an online tool to access certain information.
- **Headings**: Be sure to check your heading structures within the alerts, in respect to the page. 

### Links
- **ARIA should be used sparingly** to supplement and enhance the native features of HTML.
- **Interactive elements:** Use role="alertdialog". If the information in the alert contains interactive elements, like links or buttons, use the role="alertdialog" role instead of role="alert".

cc: @coforma-terry

**Comment by outerpress** [comment]
@laurwill the designs are updated and include the notes about pop-up blockers and logging in with the same account. Do you want to take one more look before we pass this to our developers on Wednesday?

**Comment by outerpress** [comment]
@coforma-terry I think you can close this one out. Here are our implementation tickets for reference:

Cerner redirect on the Appointments homepage - [61301](https://github.com/department-of-veterans-affairs/va.gov-team/issues/61301)
Update the static landing page - [61692](https://github.com/department-of-veterans-affairs/va.gov-team/issues/61692)
